src/auto/versions.txt

```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 83cc9ad)
+Generated by gir (https://github.com/gtk-rs/gir @ 997cbcb)
 from gir-files (https://github.com/gtk-rs/gir-files @ 22748fa)
```

Comment:
For be same as other maybe do gir-files update too, to 480c117

Reply:
Done